docs/sources/helm-charts/mimir-distributed/release-notes/v4.4.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Grafana Mimir Helm chart version 4.4 release notes"
+menuTitle: "V4.4 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 4.4"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 4.4 release notes
+
+Grafana Labs is excited to announce version 4.4 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.8 or Grafana Enterprise Metrics (GEM) v2.8. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.8](/docs/mimir/v2.8.x/release-notes/v2.8/) or [Grafana Enterprise Metrics v2.8](/docs/enterprise-metrics/v2.8.x/release-notes/v2-8/).
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and perform the necessary migrations.
+
+## Important changes
+
+- The ruler deployment max surge was changed from `0` to `50%`, and max unavailable from `1` to `0`.
+
+## Features and enhancements
+
+Notable enhancements are as follows:
+
+- **Add support for Vault Agent**
+  When enabled, the Pod annotations for TLS configurable components are updated to allow a running Vault Agent to fetch secrets from Vault and to inject them into a Pod. The annotations are updated for the following components: `admin-api`, `alertmanager`, `compactor`, `distributor`, `gateway`, `ingester`, `overrides-exporter`, `querier`, `query-frontend`, `query-scheduler`, `ruler`, `store-gateway`.
+- **Default pod security context**
+  Set default pod security context under `rbac.podSecurityContext` for easier install on OpenShift.
+- **Configure `storageClass` per zone**
+  You can now configure `storageClass` per zone for Alertmanager, StoreGateway and Ingester.
+- **Allow for deploying mixin dashboards as part of the helm chart**
```

Comment:
This is a bit hazy to me, where are these installed? I don't think we have specific documentation on this, so a bit more info would be welcome here.